modules/core/src/main/java/org/apache/ignite/spi/systemview/view/CacheView.java

```diff
@@ -344,26 +344,58 @@ public String expiryPolicyFactory() {
 
         ExpiryPolicy expiryPlc = (ExpiryPolicy)cache.cacheConfiguration().getExpiryPolicyFactory().create();
 
-        Function<Duration, String> func = (duration) -> {
-            if (duration == null)
-                return null;
+        AtomicBoolean first = new AtomicBoolean(true);
 
-            return new StringBuilder()
-                .append(duration.getDurationAmount())
-                .append(' ')
-                .append(duration.getTimeUnit() == null ? TimeUnit.MILLISECONDS : duration.getTimeUnit())
-                .toString();
-        };
+        StringBuilder expiryPlcStrBld = new StringBuilder(expiryPlc.getClass().getSimpleName());
 
-        String expiryPlcStr = new StringBuilder(expiryPlc.getClass().getSimpleName())
-            .append(" [create=").append(func.apply(expiryPlc.getExpiryForCreation()))
-            .append(", update=").append(func.apply(expiryPlc.getExpiryForUpdate()))
-            .append(", access=").append(func.apply(expiryPlc.getExpiryForAccess()))
-            .append(']').toString();
+        appendField(expiryPlcStrBld, "create", expiryPlc::getExpiryForCreation, first);
+        appendField(expiryPlcStrBld, "update", expiryPlc::getExpiryForUpdate, first);
+        appendField(expiryPlcStrBld, "access", expiryPlc::getExpiryForAccess, first);
+
+        if (!first.get())
+            expiryPlcStrBld.append(']');
 
         return S.toString((Class<Factory<?>>)cache.cacheConfiguration().getExpiryPolicyFactory().getClass(),
-            cache.cacheConfiguration().getExpiryPolicyFactory(),
-            "expiryPlc", expiryPlcStr);
+            cache.cacheConfiguration().getExpiryPolicyFactory(), "expiryPlc", expiryPlcStrBld);
+    }
+
+
+    /**
+     * @param out {@link StringBuilder} to append to.
+     * @param fieldName create/update/access expiry policy field name.
+     * @param durationSupplier Duration supplier for specified expiry policy field name.
+     * @param first {@link AtomicBoolean} flag indicating whether the field is the first in sequence.
+     */
+    private static void appendField(
+        StringBuilder out,
+        String fieldName,
+        Supplier<Duration> durationSupplier,
```

Comment:
`durationSupplier` is senseless, just use duration.

Reply:
Done!